### ISSUE TITLE
[Bug]: Your offline/check connection 

### ISSUE THREAD

**Issue by Chochochan** [YTVanced/VancedManager#776]
### Device

Device: Gaxagly 12
Operating System: youtube Vanced
Android Version:

### Version

16.29.39 version 

### Bug Description

It keep saying I'm offline or check my wifi/ network but I'm connected to it and I restarted my phone plus the app too  I did everything possible and still said the same thing 

### Additional checks

- [X] I have checked other bug reports and this is not a duplicate.
- [X] This is a bug in Vanced Manager and NOT YouTube Vanced/YouTube Vanced Music/Vanced microG.

**Comment by Qbit2** [comment]
Yeah, recently i also started getting the same error. Couldn't find a solution to this.

**Comment by Qbit2** [comment]
> Yeah, recently i also started getting the same error. Couldn't find a solution to this.

Just tried re-installimg whole thing. Downloaded install manager again from the site, and install everything again. Re-loged on microG, but even before that, YouTube loaded the content.

**Comment by Verkato** [comment]
Uninstall MicroG through the manager, reinstall MicroG and sign in with your Google account again. That resolved it for me, I suspect it was because of using a new phone and transferring old apps to it.

**Comment by NexusNovaz** [comment]
> 

This has fixed it. Thank you. I believe what you said is exactly what happened as i got a new phone not too long ago
